packages/@aws-cdk/assertions/vendor-in.sh

```diff
@@ -21,6 +21,12 @@ set -euo pipefail
 dest="lib/vendored"
 mkdir -p $dest
 
+if [[ $scriptdir != *"/packages/@aws-cdk/assertions"* ]]; then
+  # ignore errors from rsync when doing an out-of-module build
+  # needed for the build in @aws-cdk-lib
+  set +e
+fi
```

Comment:
Can you copy or re-write this comment as written in the PR description? The comment that's in the code is not clear.

Reply:
Done.